docs/vsphere/main.md

```diff
@@ -0,0 +1,40 @@
+# vSphere provider
+
+## Prerequisites
+
+1. `kubectl` CLI installed locally.
+2. vSphere instance version `6.7.0` or higher.
+3. vSphere account with aproriate priveleges.
+4. Image template.
+5. vSphere network with DHCP enabled.
+
+### Image template
+
+You can use pre-buit image templates from [CAPV project](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/main/README.md#kubernetes-versions-with-published-ovas)
+or build your own.
+
+When building your own image make sure that vmware tools and cloud-init are
```

Comment:
@a13x5 does it make sense to add links to some of the docs on how to setup vmware tools / cloud-init here?

Reply:
@DinaBelova I deliberately didn't expand this topic, because if will be end up as instruction on how to build your own image on vSphere. There are official docs, I can link it here, but they're rather broad for cloud-init without concrete steps on how to setup it, because "it depends".